### DISCUSSION TITLE
Not eligible to use Azure free account

### DISCUSSION THREAD

**Discussion by Qing-Ba** [community/community#49240] [GitHub Education]
What can GitHub Education assist you with?
I need help with a student developer pack offer

Description
I have applied for GitHub Education and bound my GitHub account with my Microsoft account, but when I applied for azure cloud service account, I was prompted that I am not qualified to use azure free account. I am a Chinese student, and I can't handle credit card for the time being. Please help solve this problem, thank you!

**Comment by samus-aran** [comment]
Hi @1172791203 👋 

We're happy you're here.

If this [Azure verification page](https://signup.azure.com/studentverification?offerType=1) does not work then we would suggest you reach out to [Azure Support](https://support.microsoft.com/en-us/topic/2315e669-8b1f-493b-5fb1-d88a8736ffe4) if you are having problems redeeming your Azure benefits.
